pkgdb/include/flox/core/util.hh

```diff
@@ -403,50 +427,48 @@ merge_vectors( const std::vector<T> & lower, const std::vector<T> & higher )
 }
 
 
+/* -------------------------------------------------------------------------- */
+
+/** @brief Convert a @a AttrPathGlob to a string for display. */
+[[nodiscard]] std::string
+displayableGlobbedPath( const AttrPathGlob & attrs );
+
+
 /* -------------------------------------------------------------------------- */
 
 /**
- * @brief Constructs a @a std::vector<std::optional<T>> from a
- * @a std::vector<T>.
+ * @brief Concatenate the given strings with a separator between
+ *        the elements.
  */
-template<typename T>
-[[nodiscard]] std::vector<std::optional<T>>
-vectorMapOptional( const std::vector<T> & orig )
+template<class Container>
+[[nodiscard]] std::string
+concatStringsSep( const std::string_view sep, const Container & strings )
```

Comment:
nonblocking: I still find the order of argument weird but

Reply:
I just align with the existing `nix` one

Reply:
ah the nix-lang one not the nix-cpp one then?
I guess makes sense to have _the list_ be the curryied argument there yes..